test/importer.spec.js

```diff
@@ -19,6 +21,65 @@ const schema = [
   }
 ]
 
+const schema2 = [
+  {
+    name: 'Title',
```

Comment:
Purely a nice to have, could we add one more that is not title, just to cover that case as well?